backend/metalit_challenge/requirements.txt

```diff
@@ -0,0 +1,8 @@
+asgiref==3.4.1
+Django==4.0
+djangorestframework==3.13.1
+mysqlclient==2.1.0
+python-dotenv==0.19.2
```

Comment:
@fadelananda for dot env library, could you change it to use environ instead? https://github.com/sloria/environs Also the implementation on settings needs to be updated too.